google/cloud/speech/cloud_speech.yaml

```diff
@@ -11,17 +11,18 @@ documentation:
     Google Cloud Speech API.
 
 apis:
-- name: google.cloud.speech.v1.Speech
+- name: google.cloud.speech.v1beta1.Speech
+- name: google.longrunning.Operations
```

Comment:
SGTM. @pongad do you have a little time on your Monday to repeat the same trick with LRO that you did with streaming?